sdk/reservations/arm-reservations/README.md

```diff
@@ -78,9 +81,12 @@ See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to
           // may cause redirects
           authManager.login();
         }
-        const client = new Azure.ArmReservations.AzureReservationAPI(res.creds, subscriptionId);
```

Comment:
Same here.

Reply:
Sorry, I cannot catch this comment. Should it be in the line `const subscriptionId = "<Subscription_Id>";` of html code?

Reply:
Yes